input/cql/CRCSMCommonFunctions.cql

```diff
@@ -0,0 +1,265 @@
+library CRCSMCommonFunctions version '1.0.0'

```

Comment:
This branch now has merge conflicts that will need to be resolved.

Probably should be done after Rute's follow-up after non-colonoscopy branch is merged in.